apps/api-server/src/routes/api/resource.js

```diff
@@ -256,14 +255,15 @@ router
 
     resourceInstance.setTags(tagIds).then((tags) => {
       // refetch. now with tags
-      let scope = [...req.scope, 'includeTags'];
+      let scope = [...req.scope];
```

Comment:
Waarom is includeTags hier weg gehaald? Mis je nu niet de tags die bij de resourceInstance horen (alleen de status tags zitten nu in de defaultscope)